services/fhir.ts

```diff
@@ -276,3 +371,13 @@ export function getConcepts(valueSetId: string, params?: SearchParams): Promise<
         params: { ...params },
     });
 }
+
+export async function applyFHIRService<T, F>(request: AxiosRequestConfig): Promise<RemoteDataResult<T, F>> {
+    return service(request);
+}
+
+export async function applyFHIRServices<T, F>(
```

Comment:
it should return Promise<RemoteDataResult<Bundle<T>, F> and it should be more complex:
you should build `{resourceType: 'Bundle', type: 'transaction|batch', entry: [for each request in requests create {resource: request.data, request: {method: request.method, ifParam: ...., url: request.url }}]}`

Reply:
and POST this object to / - it is a special endpoint for bundle